napari/layers/shapes/_shapes_list.py

```diff
@@ -1,11 +1,11 @@
 import numpy as np
-from .shape_models import Shape, Line, Path
-from .shape_utils import inside_triangles, triangles_intersect_box
-from .mesh import Mesh
-from ._constants import shape_classes, ShapeType
+from ._mesh import Mesh
+from ._shapes_models import Shape, Line, Path
+from ._shapes_utils import inside_triangles, triangles_intersect_box
+from ._shapes_constants import shape_classes, ShapeType
 
 
-class ShapeList:
+class ShapesList:
```

Comment:
Not in love with this change...? ShapeList matches LayerList which matches "car collection" which matches "card deck"...

Reply:
I will revert